blocks/library/paragraph/index.js

```diff
@@ -224,8 +226,6 @@ class ParagraphBlock extends Component {
 						/>
 					</PanelBody>
 				</InspectorControls>
-			),
-			<div key="editable">
```

Comment:
This appears useless.. but are we sure it was?

Reply:
I can revert, but I didn't discover any issues so far in my testing. There are 5ish more div wrappers there 🤣

Reply:
I will revert this particular change to be on the safe side, but I will try to remove it anyway in a separate PR when tackling this nested `div`s hell :)